accepted/2020/net5/net5.md

```diff
@@ -279,6 +279,60 @@ We'll have the following TFMs:
 |                 | (+everything else inherited from net5.0)                   |                                   |
 | Tizen, Unity... | Will follow the Xamarin model                              |                                   |
 
+### Precedences
+
+We need to decide what precedences we want the new TFMs to have in relation to
+the old ones. For example, consider this scenario:
+
+* Project: `net5.0-ios`
+* Package:
+  * `net5.0`
+  * `xamarin.ios`
+
+Both `net5.0` and `xamarin.ios` are applicable for the project. Which assets
+should NuGet prefer?
+
+There are effectively two options:
+
+1. Treat existing Xamarin TFMs as more specific
+2. Treat `net5.0` as more specific
+
+It seems easier to explain that `net5.0-android` and `net5.0-ios`  are simply
+renamed versions of `xamarin.android` and `xamarin.ios` respectively.  Both are
+platform specific. `net5.0` is a portable TFM, similar to `netcoreapp` and
+`netstandard`. It seems generally more logical to first look for platform
+specific TFMs and then fall back to portable TFMs:
+
+1. First look for the same TFM
```

Comment:
fwiw, I don't think there's any mapping that does otherwise :)

Reply:
Yep, just listed for completeness :-)